src/Whispr.EntityFrameworkCore/Cleaning/OutboxCleanupService.cs

```diff
@@ -44,16 +44,16 @@ private async ValueTask CleanupOutboxMessages(CancellationToken cancellationToke
         {
             var expiredUtc = DateTimeOffset.UtcNow - _retentionPeriod;
             var rowsDeleted = await dbContext.Set<OutboxMessage>()
-                .Take(_batchSize)
```

Comment:
This translates to a SELECT TOP(X) so order isn't really important, but it is cleaner indeed!